src/ray/gcs/redis_context.cc

```diff
@@ -161,8 +178,9 @@ Status RedisContext::AttachToEventLoop(aeEventLoop *loop) {
 }
 
 Status RedisContext::RunAsync(const std::string &command, const UniqueID &id,
-                              uint8_t *data, int64_t length, const TablePrefix prefix,
-                              const TablePubsub pubsub_channel, int64_t callback_index) {
+                              const uint8_t *data, int64_t length,
+                              const TablePrefix prefix, const TablePubsub pubsub_channel,
```

Comment:
Nit: I usually don't const enums which just get compiled to int.  So const has no meaningful semantics in this case, I think.  Up to you.